test/test_console_tool.py

```diff
@@ -227,16 +227,7 @@ def test_buckets(self):
 
         # Delete one
         expected_stdout = '''
-        {{
-            "accountId": "{account_id}",
-            "bucketId": "bucket_1",
-            "bucketInfo": {{}},
-            "bucketName": "your-bucket",
-            "bucketType": "allPrivate",
-            "corsRules": [],
-            "lifecycleRules": [],
-            "revision": 1
-        }}
+        null
```

Comment:
@bwbeach do we bump b2cli version from 1.4.x to 1.5.0 due to this change potentially breaking interface? Nobody should be using this anyway, the return contents is not really documented, that's why it was removed... But I feel we should increase middle version. How do you think?